crates/yakui-winit/src/lib.rs

```diff
@@ -53,11 +50,7 @@ impl YakuiWinit {
         self.auto_viewport = enabled;
     }
 
-    pub fn handle_event<T>(
-        &mut self,
-        state: &mut yakui_core::Yakui,
-        event: &WinitEvent<T>,
-    ) -> bool {
+    pub fn handle_event(&mut self, state: &mut yakui_core::Yakui, event: &WindowEvent) -> bool {
```

Comment:
We should preserve yakui's ability to handle winit's `Event` type to make it easier for users still using the old event loop style to integrate. Additionally, we probably do want to leave the door open to handle `DeviceEvent` without silently missing out on functionality.

A good way to solve this might be with an `AsWinitEvent` trait that we implement for `Event`, `DeviceEvent`, and `WindowEvent`. I'm not sure what the exact shape it should take is.

Reply:
we were more thinking of having a separate function for device events if and when it's needed